vscode/language-configuration.json

```diff
@@ -19,5 +19,6 @@
     ["[", "]"],
     ["(", ")"],
     ["{", "}"]
-  ]
+  ],
+  "wordPattern": "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\@\\#\\%\\^\\&\\*\\(\\)\\-\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\?\\s]+)"
```

Comment:
What was this added for? What was the issue?

Reply:
Hmmm. So, word boundaries influence how completions behave in VS Code. For example, completion gets triggered when you start a new word. Or VS Code uses word boundaries to figure out which range it should replace when you commit a completion. (see: https://github.com/microsoft/vscode/blob/b114662b7afdb3823726bdc8de8d31d17d1588a9/src/vscode-dts/vscode.d.ts#L4516)

But I can't recall what issue I was running into that made me add this. The default should work. I'll try removing it and see what happens.

Reply:
Reverting this for now - I don't think it does anything, and completions are half-baked right now anyway.